### PR TITLE
[14.0][ADD] stock_picking_tier_validation

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module extends the functionality of Transfers to support a tier validation process.

### CODE REVIEW

**Review by RLeeOSI — APPROVED** [review]
LGTM. We've been using this with no issues

File: stock_picking_tier_validation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "version": "14.0.1.0.0",
+    "category": "Warehouse Management",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["stock", "base_tier_validation"],
+    "data": ["views/stock_picking_views.xml"],
+    "installable": True,
+    "development_status": "Alpha",
+}
```
Comment (kittiu) [inline stock_picking_tier_validation/__manifest__.py:17]: Pleas add maintainers

**Review by kittiu — APPROVED** [review]

**Comment by houssine78** [comment]
Hi @dreispt could you rebase trough the oca bot command ?

**Review by houssine78 — APPROVED** [review]

**Pull request merged**